tools/autodeploy/templates/minimal_example/Makefile.template

```diff
@@ -1,4 +1,16 @@
-rootdir := ../../../..
+# Initialize rootdir to be cwd
```

Comment:
This assumes that your target dir is inside neuralSPOT. When I ran from outside my neuralspot dir it crashed my Mac:

ns_autodeploy --tflite-filename ../../ourmodels/kws/kws_ref_model_aligned.tflite --verbosity 4 --makefile-directory ../neuralSPOT --onboard-perf

I know why you did this: the neuralSPOT version of minimal-example is broken in that it actually depends on neuralSPOT when it should be completely independent and compile anywhere. I fixed that in ns-mirror.

For now, lets call this an errata - add a check to ns_autodeploy to ensure they are running inside neuralSPOT and error out if they're not and minimal example is being created.